src/sage/graphs/graph.py

```diff
@@ -4901,16 +4901,24 @@ def independent_set_of_representatives(self, family, solver=None, verbose=0,
         return repr
 
     @doc_index("Algorithmically hard stuff")
-    def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3):
+    def minor(self, H, solver=None, verbose=0, induced=False, *, integrality_tolerance=1e-3):
         r"""
-        Return the vertices of a minor isomorphic to `H` in the current graph.
+        Return the vertices of a minor isomorphic to `H` in the current graph. 
```

Comment:
you must remove the trailing whitespace at the end of the line.

Sorry for the noise.